build.gradle

```diff
@@ -1,3 +1,5 @@
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
```

Comment:
back to my Intellij error which I never was able to fix

Reply:
:joy: will revert that change in next PR